Userland/Demos/CatDog/CatDog.cpp

```diff
@@ -10,174 +10,187 @@
 #include <LibGUI/Painter.h>
 #include <LibGUI/Window.h>
 
+namespace {
+
+enum State : uint16_t {
```

Comment:
Please don't use `<stdint.h>` in new Serenity code. `AK/Types.h` types such as u16 are preferred.

That being said, for this enum, It would be preferred to use an `enum class` and then apply  the `AK_ENUM_BITWISE_OPERATORS` macro to the class from `AK/EnumBits.h`.

Reply:
Thank you for pointing out the AK EnnumBits.
I've promoted State to a "real" enum and switched to using the has_flag functions.